src/main/java/bio/terra/service/resourcemanagement/azure/AzureResourceConfiguration.java

```diff
@@ -109,4 +118,53 @@ public void setHomeTenantId(UUID homeTenantId) {
       this.homeTenantId = homeTenantId;
     }
   }
+
+  public static class Synapse {
+
+    private String workspaceName;
+    private String sqlAdminUser;
+    private String sqlAdminPassword;
+    private String databaseName;
+    private String parquetFileFormatName;
+
+    public String getWorkspaceName() {
+      return workspaceName;
+    }
+
+    public void setWorkspaceName(String workspaceName) {
+      this.workspaceName = workspaceName;
+    }
+
+    public String getSqlAdminUser() {
+      return sqlAdminUser;
+    }
+
+    public void setSqlAdminUser(String sqlAdminUser) {
+      this.sqlAdminUser = sqlAdminUser;
+    }
+
+    public String getSqlAdminPassword() {
+      return sqlAdminPassword;
+    }
+
+    public void setSqlAdminPassword(String sqlAdminPassword) {
+      this.sqlAdminPassword = sqlAdminPassword;
+    }
+
+    public String getDatabaseName() {
+      return databaseName;
+    }
+
+    public void setDatabaseName(String databaseName) {
+      this.databaseName = databaseName;
+    }
+
+    public String getParquetFileFormatName() {
```

Comment:
Is this something that will change from environment to environment?

Reply:
Since this variable is created manually via query to the synapse database, I want to be able to easily change this on the fly. But, yes, in theory it can also eventually change between environments and central synapse instances.